nncf/onnx/hardware/fused_patterns.py

```diff
@@ -52,8 +52,12 @@ def _get_onnx_hw_fused_patterns() -> HWFusedPatterns:
                                          activations + batch_norms | \
                                          batch_norms | activations
 
+    batch_norm_arithmetic_arithmetic_act = batch_norms + arithmetic_ops + arithmetic_ops + activations
```

Comment:
Let's add new patterns in the proper and clean way: with the particular method, `add_node` and `add_edge` functionality, for each node, with the needed type (not **ARITHMETIC_OPS). It's more convenient to support, readable, and easier to imagine/describe graphically. 
For now, you've added the A*B*C*D = E numbers of patterns, while initially describing only one that is really needed.
Additionally, the original pattern looks like a `batch norm layer + scale shift pattern` and not as `BN_ARITHMETHIC_ARITHMETHIC_ACT`, `LINEAR + BN_ARITHMETHIC_ARITHMETHIC_ACT_`, ..., etc.

Reply:
I agree with you that `add_node` and `add_edge` is more explicit way to create patterns.
At the same time there are two patterns that should be added: 
BN + MUL + ADD + ACTIVATION and LINEAR + MUL + ADD +ACTIVATION, as both of them are faced in the model

![image](https://user-images.githubusercontent.com/32935044/213743108-cb8f1ec4-8b50-4695-bb27-916ea3ea867c.png)

Reply:
You can add only one pattern like:
```suggestion
pattern = GraphPattern()
batchnorm_node = pattern.add_node(GraphPattern.LABEL_ATTR: 'BATCHNORM', GraphPattern.METATYPE_ATTR: om.ONNXBatchNormMetatype)
mul_node = pattern.add_node(GraphPattern.LABEL_ATTR: 'MULTIPLY', GraphPattern.METATYPE_ATTR: om.ONNXMulLayerMetatype)
add_node = pattern.add_node(GraphPattern.LABEL_ATTR: 'ADD', GraphPattern.METATYPE_ATTR: om.ONNXAddLayerMetatype)
pattern.add_edge(batchnorm_node, mul_node)
pattern.add_edge(mul_node, add_node)
```
instead of the overburdened and unreadable list of `BN + MUL + ADD + ACTIVATION` and `LINEAR + MUL + ADD +ACTIVATION`

Reply:
Again - I need two patterns: `linear_ops + batch_norm + scale_shift + activations` and `batch_norm + scale_shift + activations`
I propose to use the already created `scale_shift` pattern and using `+` operator to connect them.
`+` - is easy understandable operation

Reply:
If you need two patterns, then add two patterns with the two different methods with different labels, instead of using the `+` operators.

Reply:
As you wish. I'm not insisting.

Reply:
I hope other reviewers will add their minds here and we can make a finall decision whether should we add patterns only with `add_node` and `add_edge` methods.

Reply:
I've updated the patterns without + and | operators.